Sources/SpeziOnboarding/OnboardingConsentView.swift

```diff
@@ -73,25 +92,42 @@ public struct OnboardingConsentView: View {
                     .padding(.bottom)
                 },
                 actionView: {
-                    OnboardingActionsView(
-                        LocalizedStringResource("CONSENT_ACTION", bundle: .atURL(from: .module)),
+                    AsyncButton(
```

Comment:
Can probably just be a normal `Button` as we don't do any asynchronous actions here.